### ISSUE TITLE
Script outputs emoticons instead of emoji.

### ISSUE THREAD

**Issue by alexanderpas** [chris-rands/emojify#2]
The script outputs Emoticons `;)` instead of Emoji `😉`. Emoji can also be used in python code

The difference:

- https://en.wikipedia.org/wiki/Emoticon
- https://en.wikipedia.org/wiki/Emoji

Additionally, this means the list() language construct can be used when all spaces are removed, since all emoji characters are 1 unicode character in length, this will reduce the length of the output script by a significant amount.

**Comment by chris-rands** [comment]
Thank you for your comments. I used the term "emoji icons" but I agree the semantics were imprecise. I have updated the README and the code to mention emoticons. I will not change the script or repo name since there may be changes to the code in the future but I will leave this issue open for now.

I want to keep the functionality with emoticons but I would welcome the use of real emojis as an additional layer. The reasons to keep the current way is that first sometimes a simple ASCII encoding might be desirable. Second, the current implementation is generic, it can encode a python script with virtually any 10 character alphabet.

On the use of real emojis, I may implement this myself when I have time or I would welcome a good PR. This should be an additional option that is switched on/off via `argparse` arguments. The documentation and unit tests would also need updating of course. On the point of reducing the output script size, this was never a major concern ;). Moreover, I am not convinced this works, because to keep the full functionality there would need to be an emoji for *every* unicode character.
